rust/osm2lanes/Cargo.toml

```diff
@@ -1,18 +1,30 @@
 [package]
 name = "osm2lanes"
-version = "0.1.0"
+version = "0.1.0-alpha.1"
 edition = "2021"
 authors = ["Dustin Carlino <dabreegster@gmail.com>", "Michael Droogleever Fortuyn <droogmic@gmail.com>"]
 license = "Apache 2.0"
+description = "Transform tags from an OpenStreetMap (OSM) way into a description of the lanes"
+repository = "https://github.com/a-b-street/osm2lanes/"
+
+[lib]
+# cdylib for WASM
+crate-type = ["lib", "cdylib"]
 
 [dependencies]
+console_error_panic_hook = { version = "0.1", optional = true }
 log = "0.4"
-serde = { version = "1", features = ["derive"] }
 reqwest = { version = "0.11", optional = true, features = ["blocking", "json"] }
+serde = { version = "1", features = ["derive"] }
+wasm-bindgen = { version = "0.2", optional = true, features = ["serde-serialize"] }
+wee_alloc = { version = "0.4", optional = true }
 
 [features]
+default = ["console_error_panic_hook"]
```

Comment:
Wait, why's this in default? Related to trying to make features per compilation target work?